purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesConfiguration.kt

```diff
@@ -80,33 +80,21 @@ open class PurchasesConfiguration(builder: Builder) {
         }
 
         /**
-         * **Feature in beta**. Enables signature verification of requests to the RevenueCat backend
-         * in informational mode and enables diagnostics reports to RevenueCat to help us analyze this feature.
-         * Informational mode means that the behavior will remain the same but we will provide information about
-         * detected attempts of hacking.
+         * Enables signature verification of requests to the RevenueCat backend
+         * and enables diagnostics reports to RevenueCat to help us analyze this feature.
          *
          * When changing from disabled to enabled, the SDK will clear the CustomerInfo cache.
          * This means users will need to connect to the internet to get their entitlements back.
          *
          * The result of the verification can be obtained from [EntitlementInfos.verification] or
          * [EntitlementInfo.verification].
          *
-         * This feature is currently in beta and the behavior may change. Only available in Kotlin.
-         *
          * Default mode is disabled.
-         *
-         * @warning This function is marked as [ExperimentalPreviewRevenueCatPurchasesAPI] and may change in the future.
          */
         @JvmSynthetic
-        @ExperimentalPreviewRevenueCatPurchasesAPI
-        fun informationalVerificationModeAndDiagnosticsEnabled(enabled: Boolean) = apply {
-            if (enabled) {
-                this.verificationMode = EntitlementVerificationMode.INFORMATIONAL
-                this.diagnosticsEnabled = true
-            } else {
-                this.verificationMode = EntitlementVerificationMode.DISABLED
-                this.diagnosticsEnabled = false
-            }
+        fun verificationModeAndDiagnostics(mode: EntitlementVerificationMode) = apply {
```

Comment:
I think this should be the final API, so it can support `ENFORCED` when we re-add that, and so that it provides a consistent API for the hybrids. Thoughts @tonidero?

Reply:
Well, In Android, I changed the parameter to a boolean since it kinda made more sense to me that, for this function that enabled 2 features to be a boolean instead of an `EntitlementVerificationMode` which doesn't have anything to do with diagnostics. I do agree that for the final release, if we separate both features, passing an enum like this to configure trusted entitlements makes the most sense. Happy to chat more about this though! cc @aboedo In case you have any thoughts.

Another question is, do we want to tie diagnostics to trusted entitlements for the final release? IMO, I mostly would like these features tied only while they are on beta so we can be more confident about it until we release it fully. I would split them after that.